### ISSUE TITLE
Issue causing Android to crash. 

### ISSUE THREAD

**Issue by rklus16** [jeduan/cordova-plugin-facebook4#198]
I've noticed on Android this error has caused crashes.  It's only been a handful of crashes but only error shown causing crashes. I was wondering if I'm not accounting for some edge case?  I haven't been able to replicate the crash.

```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'void org.apache.cordova.CallbackContext.success(org.json.JSONObject)' on a null object reference
       at org.apache.cordova.facebook.ConnectPlugin$2$1.onCompleted(ConnectPlugin.java:116)
       at com.facebook.GraphRequest$1.onCompleted(GraphRequest.java:304)
       at com.facebook.GraphRequest$5.run(GraphRequest.java:1379)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:145)
       at android.app.ActivityThread.main(ActivityThread.java:5951)
       at java.lang.reflect.Method.invoke(Method.java)
```

**Comment by Heshyo** [comment]
Maybe it's the same as https://github.com/jeduan/cordova-plugin-facebook4/issues/144

**Comment by robertarnesson** [comment]
should be fixed @0a5ac31
